### PR TITLE
Use uri in App

### PR DESCRIPTION
**Pull request by djmetzle**
Use the ApiUri object in the App endpoints.

### CODE REVIEW

**Comment by g1y** [comment]
CR :+1:

File: lib/api/app/app.rb [hunk 1]
```diff
@@ -25,16 +25,21 @@ def list_license_fields(appid:)
 	end
 
 	def create_license_field(appid:, options:)
+		method = "POST"
 		endpoint = "app/" + appid + "/licensefield"
-		return self.client.request("POST", endpoint, options)
+		uri = ApiUri::build_uri(endpoint)
+		return self.client.request(method, endpoint, options)
```
Comment (g1y) [inline lib/api/app/app.rb]: shouldn't this be uri

**Comment by djmetzle** [comment]
un_dev_block :exclamation:

**Comment by g1y** [comment]
Merge this shit URI day

**Comment by g1y** [comment]
CR :+1:

**Pull request merged**